src/classes/LineItemTriggerTest.cls-meta.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <ApexClass xmlns="http://soap.sforce.com/2006/04/metadata">
-    <apiVersion>31.0</apiVersion>
+    <apiVersion>30.0</apiVersion>
```

Comment:
This component uses an obsolete API version. Please upgrade to Spring '17 (39.0) or later.